userspace/libsinsp/CMakeLists.txt

```diff
@@ -200,11 +200,10 @@ endif()
 set_sinsp_target_properties(sinsp)
 
 target_link_libraries(sinsp
-		PUBLIC scap
+		PUBLIC scap "${RE2_LIB}"
```

Comment:
This is needed so that unit-test-libsinsp is able to correctly link `re2` since it is now needed.